cli/stork-publisher-agent/websocket.go

```diff
@@ -149,8 +166,24 @@ func (owc *OutgoingWebsocketConnection[T]) Writer() {
 				logger.Warn().Msg("attempted to send message on closed websocket")
 				return
 			}
+
+			filteredPriceUpdates := make(SignedPriceUpdateBatch[T])
+			owc.assetIdsLock.RLock()
+			_, allAssets := owc.assetIds[WildcardSubscriptionAsset]
+			if allAssets {
+				filteredPriceUpdates = signedPriceUpdateBatch
```

Comment:
This is where the asset filtering happens - if the configuration changes in the registry we'll update assetIds and the filtering will happen without needing to touch the existing connection